pombola/south_africa/management/commands/south_africa_import_new_constituency_office_locations.py

```diff
@@ -0,0 +1,138 @@
+"""Command to import PMG's constitiency office data
+
+PMG are undertaking an exercise to visit all the constituency
+offices and collect accurate locations and photographs.
+The data from this is available at
+
+https://app.m4jam.com/app/campaigns/2298/export/
+
+This script parses the resulting CSV file and updates our data
+on constituency offices to have the correct location.
+"""
+
+
+import csv
+import os.path
+import re
+
+import requests
+
+from django.conf import settings
+
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+
+from pombola.core.models import (
+    Organisation,
+    )
+from pombola.core.utils import mkdir_p
+
+from pombola.images.models import Image
+
+
+# Matches the format of the locations in the CSV we get from PMG.
+location_re = re.compile(
+    r'SRID=4326;POINT\(-?([0-9]+\.[0-9]+) (-?[0-9]+\.[0-9]+)\)')
```

Comment:
The first group doesn't include the `-` if it's present. (The longitude won't be negative in South Africa, but this might get reused elsewhere.)